config/apps/argo-server.yaml

```diff
@@ -123,6 +123,10 @@ spec:
             scheme: HTTP
           initialDelaySeconds: 10
           periodSeconds: 20
+        resources:
+          requests:
+            cpu: 100m
+            memory: 20Mi
```

Comment:
seems pretty low? how come 20Mi?

Reply:
I looked at the controller memory usage. it is using 18Mi max  with two informer (pipeline + step).  Currently, Controller is configuring below resource Quota.  I think `20Mi` is good for request.
```yaml
        resources:
          limits:
            cpu: 100m
            memory: 30Mi
          requests:
            cpu: 100m
            memory: 20Mi
```